src/sentry/api/endpoints/group_autofix_setup_check.py

```diff
@@ -44,7 +45,7 @@ def get_autofix_integration_setup_problems(
 
     organization_integration = organization_integrations[0] if organization_integrations else None
     integration = organization_integration and integration_service.get_integration(
-        organization_integration_id=organization_integration.id
+        organization_integration_id=organization_integration.id, status=ObjectStatus.ACTIVE
```

Comment:
this should query for active because downstream github API calls will not go through if the integration isn't active